_posts/2017-03-14-pycon-pune-2017-a-wonderful-python-experience.md

```diff
@@ -0,0 +1,220 @@
+---
+layout: post
+title: "Pycon Pune 2017: A wonderful Python conference"
+date: "2017-03-14 18:18:02 +0530"
+tag: python conference
+---
+
+![pycon_pune_group_photo]({{
+site.url}}/assets/images/pycon_pune_group_photo.jpg)
+
+## tl;dr
+
+The conference is worth to attend if you are a student, programmer or a
+hobbyist.  You are swag-hungry then don't expect much as a swag from this
+conference. You are a Devsprint lover, then this conference has the coolest
+Devsprint. A great number of keynotes speackers are invited for this conference.
+
+
+## Detailed Experience
+
+Because I was volunteering for this conference I reached Pune one day earlier
+from the conference days. The volunteer meeting was happening at Reserved-bit.
+
+### Volunteer
+
+#### Reserved-bit
+
+[Reserved-bit][reservedbit] is the best hackerspace I have ever come across. It
+has a large collection of programmable boards. You will find boards like
+[Raspberry Pi](https://en.wikipedia.org/wiki/Raspberry_Pi), [Bana
+Pi](https://en.wikipedia.org/wiki/Banana_Pi),
+[Dragonboard](https://developer.qualcomm.com/hardware/dragonboard-410c),
+[Bigalbon](https://www.bigalsmods.com/product-p/1001.htm),
+[BBC-microbit](https://en.wikipedia.org/wiki/Micro_Bit) and the 3D printer. More
+on, this space has a great collection of books on Compilers and Embedded
```

Comment:
"More on" sounds a bit strange. Consider "Furthermore", "Moreover", "In addition"